plugins/frontend/backstage-plugin-cloudsmith/README.md

```diff
@@ -0,0 +1,13 @@
+# backstage-plugin-cloudsmith
+
+To use this plugin, you will need to add the cards to the entity page in the frontend app.
+
+Edit the `packages/app/src/components/home/Homepage.tsx` and add the imports
+
+Then add the following components:
+
+```typescript jsx
+<Grid item md={6}>
+  <CloudsmithStatsCard repo="repo" owner="owner" />
+</Grid>
+```
```

Comment:
Can you please be more descriptive in the README ? This is going to be the only documentation about this plugin. 
Can you add pls images of the plugin ? How will it look like if configured properly ? 
Can you please add a section about that they'll need to configure a proxy for it to work ? 
Can you please explain what is 'repo' and 'owner' ?
Can you please mention that they'll need to have homepage enabled (it is not a default feature of backstage still.) And link to the docs how to do it?